ui/components/Modals/PublishModal.js

```diff
@@ -1,26 +1,56 @@
-import React from "react";
-import { publish_schema, publish_ui_schema } from "../schemas/publish_schema";
+import React, { useState, useEffect } from "react";
 import Modal from "../Modal";
-import PublicIcon from '@material-ui/icons/Public';
+import PublicIcon from "@material-ui/icons/Public";
+import _ from "lodash";
+import { getMeshModels } from "../../lib/api/meshmodel";
```

Comment:
Did you accidentally leave this file out?

Build failure:

```
Failed to compile.

./components/Modals/PublishModal.js
Module not found: Can't resolve '../../lib/api/meshmodel' in '/Users/lee/code/meshery/ui/components/Modals'

Import trace for requested module:
./components/Modals/PublishModal.js
./components/NavigatorExtension.js


> Build failed because of webpack errors
cd provider-ui; npm run build && npm run export; cd ..
```

Reply:
path should be "../../api/meshmodel" instead of "../../lib/api/meshmodel"

Fixed